modules/basegl/src/processors/volumeprocessing/volumeregionshrink.cpp

```diff
@@ -181,11 +226,21 @@ void VolumeRegionShrink::process() {
 
     utilgl::Activate as{&shader_};
 
+    if (volumeNumericType_ == "i") {
+        shader_.setUniform("fillValue", ivec4{fillValue_.get()});
+    } else if (volumeNumericType_ == "u") {
+        shader_.setUniform("fillValue", static_cast<uvec4>(fillValue_.get()));
```

Comment:
why the static cast of `uvec4` and not a static_cast of the value?
`uvec4{static_cast<std:;uint32_t>(fillValue_.get())}`